include/tc/core/polyhedral/schedule_tree_elem.h

```diff
@@ -18,6 +18,7 @@
 #include <memory>
 #include <unordered_set>
 #include <vector>
+#include <sstream>
```

Comment:
can you move this in alphabetical order? clang-format might complain

Reply:
s/may/will :)

Reply:
Yeah, clang format is a pitta for syncing files with fbcode due to config differences :( We have this problem in Caffe2 as well. Hopefully we will find a way to sync. clang configs for OSS and fbcode. 

You can imagine number of lint errors I see when I imported the whole code base in another direction :)

Reply:
yeah, as I mentioned, it is a problem. As your oss clang might just be of a different version than our internal. And then configs might be non compatible. And I agree, we need to make sure clang config for OSS TC, Caffe2 and PyTorch is the same :)

Reply:
Our CI runs clang-format-5.0 (the same we have prebuilt in docker).